src/operator/context/operator.tsx

```diff
@@ -121,6 +128,13 @@ export const OperatorProvider: FC<Props> = React.memo(({children}) => {
     status = RemoteDataState.Loading
   }
 
+  const hasWritePermissions = () => {
+    return (
+      !isFlagEnabled('operatorRole') ||
+      (quartzMe.isOperator && quartzMe?.operatorRole === 'read-write')
+    )
+  }
```

Comment:
this should never change right? If that's the case we should just make this an inferred value rather than a function that's invoked:

```
const hasWritePermissions = !isFlagEnabled('operatorRole') ||
      (quartzMe.isOperator && quartzMe?.operatorRole === 'read-write')